### ISSUE TITLE
How is iterations per epoch set for JEPA pre-training?

### ISSUE THREAD

**Issue by NoTody** [facebookresearch/vjepa2#91]
Hi, @mmuckley ,

I'm quite confused about how iterations per epoch (ipe=300) and ipe_scale=1.25 are set in the provided config files. Did you just empirically find the optimal number of iterations per epoch and scaling factor or are there any specific reasons why JEPA pre-training is not passing through all samples for each epoch?

**Comment by mmuckley** [comment]
Hi @NoTody, "epoch" in the code is actually somewhat meaningless. It's really just a tracker for when to save checkpoints and refresh the dataloader.

`ipe_scale` adjusts the various schedules (learning rate, weight decay) to compress or decompress the overall training cycle. The convention to use 1.25 is ultimately just inherited directly from [V-JEPA](https://github.com/facebookresearch/jepa/blob/51c59d518fc63c08464af6de585f78ac0c7ed4d5/configs/pretrain/vitl16.yaml#L79).
